### PR TITLE
Modified the description of the config setting lookupCenterNameUsing

### PR DESCRIPTION
**Pull request by cmadjar**
Modified the description of the config setting lookupCenterNameUsing to something more accurate and meaningful.

See also: 
- PR on the LORIS-MRI side: https://github.com/aces/Loris-MRI/pull/291
- redmine ticket: https://redmine.cbrain.mcgill.ca/issues/14088

FYI: setting the milestone to 19.1 (same milestone as the LORIS-MRI one)

### CODE REVIEW

File: SQL/Archive/2018-04-06_ImprovedDescriptionOfLookupCenterNameUsing.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-- Change description and label of the ConfigSetting named lookupCenterNameUsing
+UPDATE ConfigSettings SET
+    Description = "DICOM field to use to get the patient and center name of the DICOM study, being either PatientID or PatientName",
```
Comment (MounaSafiHarab) [inline SQL/Archive/2018-04-06_ImprovedDescriptionOfLookupCenterNameUsing.sql]: I would rephrase to say "the patient identifier (being either the PatientName or PatientID), and center name...."

Comment (cmadjar) [reply SQL/Archive/2018-04-06_ImprovedDescriptionOfLookupCenterNameUsing.sql]: aaaaah, I was looking for that "identifier" word but it never came to my brain... Thank you!! Changing it right away :)

File: SQL/0000-00-03-ConfigTables.sql [hunk 2]
```diff
@@ -141,7 +141,7 @@ INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType,
...
 INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'tarchiveLibraryDir', 'Location of storing the library of used tarchives. If this is not set, they will not be moved', 1, 0, 'text', ID, 'Path to Tarchives', 8 FROM ConfigSettings WHERE Name="imaging_pipeline";
-INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'lookupCenterNameUsing', 'DICOM field to use to get the patient and center name of the DICOM study, being either PatientID or PatientName', 1, 0, 'text', ID, 'Patient and center name lookup variable', 9 FROM ConfigSettings WHERE Name="imaging_pipeline";
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'lookupCenterNameUsing', 'DICOM field to use to get the patient identifiers (being stored either in PatientName or PatientID) and the center name of the DICOM study', 1, 0, 'text', ID, 'Patient identifiers and center name lookup variable', 9 FROM ConfigSettings WHERE Name="imaging_pipeline";
```
Comment (driusan) [inline SQL/0000-00-03-ConfigTables.sql]: Since this is a text field, maybe it should also say what the valid values are?

(But I'm still willing to merge this as-is if you want, since it's an improvement over the current description..)

Comment (cmadjar) [reply SQL/0000-00-03-ConfigTables.sql]: I just updated the description so that it is clearer (hopefully). Let me know what you think or if you think of a better way to make this clearer. Thanks!
